cronjob/templates/cronjob.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ .Chart.Name }}
+spec:
+  schedule: {{ .Values.schedule | quote }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: {{ .Values.image.name }}
+            appVersion: {{ .Values.image.tag }}
```

Comment:
we should have the following labels here

```
jobTemplate:
  spec:
    template:
      metadata:
        labels:
          app: {{ template "cronjob.name" . }}
          release: "{{ .Release.Name }}"
```

Reply:
https://github.com/commercetools/k8s-charts/pull/1/commits/3ee7bd2cba21e037cef66c8efb078cf686a27d0e